test/sql/test_alter_table/R/test_alter_table_abnormal

```diff
@@ -281,4 +281,19 @@ ALTER TABLE site_access1 DROP COLUMN site_id;
 -- result:
 E: (1064, 'Distribution column[site_id] cannot be dropped. index[site_access1]')
 -- !result
+-- name: test_alter_key_buckets
```

Comment:
need to delete

Reply:
Had changed to optimize table，Thanks~ @meegoo